pkgs/build-support/bintools-wrapper/default.nix

```diff
@@ -59,6 +59,7 @@
 , postLinkSignHook ? null, signingUtils ? null
 }:
 
+#assert propagateDoc -> bintools ? man;
```

Comment:
intentional?

Reply:
I meant to uncomment that! I ran into errors during development that were drastically simplified by this assertion. I've pushed a fix.